apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-editor-nux/src/welcome-tour/tour-launch.tsx

```diff
@@ -20,33 +20,38 @@ function LaunchWpcomWelcomeTour() {
 			select( 'automattic/starter-page-layouts' ).isOpen(),
 		isManuallyOpened: select( 'automattic/wpcom-welcome-guide' ).isWelcomeGuideManuallyOpened(),
 	} ) );
+	const { siteIntent, siteIntentFetched } = useSiteIntent();
 	const localeSlug = useLocale();
 
 	// Preload first card image (others preloaded after open state confirmed)
-	usePrefetchTourAssets( [ getTourSteps( localeSlug, false )[ 0 ] ] );
+	usePrefetchTourAssets( [ getTourSteps( localeSlug, false, false, null, siteIntent )[ 0 ] ] );
 
 	useEffect( () => {
 		if ( ! show && ! isNewPageLayoutModalOpen ) {
 			return;
 		}
 
+		if ( ! siteIntentFetched ) {
+			return;
+		}
```

Comment:
I changed the `useSiteIntent`'s interface because `recordTracksEvent` will be fired twice; when it's in the default state and when it finishes fetching site intent. 
I thought we need to have `siteIntentFetched`, but any thoughts?